idf_build_apps/manifest/manifest.py

```diff
@@ -232,8 +277,16 @@ def enable_test_targets(
     ) -> t.List[str]:
         return self._most_suitable_rule(folder).enable_test_targets(default_sdkconfig_target, config_name)
 
-    def depends_components(self, folder: str) -> t.List[str]:
-        return self._most_suitable_rule(folder).depends_components
+    def depends_components(
+        self, folder: str, default_sdkconfig_target: t.Optional[str] = None, config_name: t.Optional[str] = None
```

Comment:
```suggestion
        self, folder: str, target: t.Optional[str] = None, config_name: t.Optional[str] = None
```

to keep the naming consistency with others

Reply:
Agree, that's why I chose to use the name "default_sdkconfig_target."
```python
 def enable_build_targets(
        self, folder: str, default_sdkconfig_target: t.Optional[str] = None, config_name: t.Optional[str] = None
    ) -> t.List[str]:
        return self._most_suitable_rule(folder).enable_build_targets(default_sdkconfig_target, config_name)

    def enable_test_targets(
        self, folder: str, default_sdkconfig_target: t.Optional[str] = None, config_name: t.Optional[str] = None
    ) -> t.List[str]:
        return self._most_suitable_rule(folder).enable_test_targets(default_sdkconfig_target, config_name)
```

Reply:
BTW, I renamed it to "target" in two functions. If you also think it's better to keep the name "default_sdkconfig_target," I can revert the changes.

Reply:
oh, sorry for misleading you. my bad :(

yes you're right. I was comparing it to some other functions. could you help revert them to `default_sdkconfig_target` instead? Thanks a lot.